### PR TITLE
SQL examples for various SDKs - set 1 

### PR DESCRIPTION
**Pull request by Vandana-Rajamani**
Added new examples for all SDKs:

Directories impacted:
nosql-examples/examples-nosql-java-sdk
nosql-examples/examples-nosql-python-sdk
nosql-examples/examples-nosql-go-sdk
nosql-examples/examples-nosql-node-sdk
nosql-examples/examples-nosql-dotnet-sdk

Signed-off-by:Vandana Rajamani vandana.rajamani@oracle.com

### CODE REVIEW

File: examples-nosql-dotnet-sdk/sqlexamples/AddData.cs [hunk 1]
```diff
@@ -0,0 +1,342 @@
+/*-
+ * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (mikebrey) [inline examples-nosql-dotnet-sdk/sqlexamples/AddData.cs:2]: change 2021 to 2023

File: examples-nosql-dotnet-sdk/sqlexamples/AlterTable.cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
+/*-
+ * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (mikebrey) [inline examples-nosql-dotnet-sdk/sqlexamples/AlterTable.cs:2]: change 2021 to 2023

File: examples-nosql-dotnet-sdk/sqlexamples/CreateTable.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
+/*-
+ * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (mikebrey) [inline examples-nosql-dotnet-sdk/sqlexamples/CreateTable.cs:2]: change 2021 to 2023

File: examples-nosql-go-sdk/sqlexamples/AddData.go [hunk 4]
```diff
@@ -0,0 +1,334 @@
+package main
```
[the remark this answers is not in the document]
Comment (mikebrey) [reply examples-nosql-go-sdk/sqlexamples/AddData.go:1]: this appears to be a brand new file , make year 2022, 2023

File: examples-nosql-java-sdk/sqlexamples/AlterTable.java [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+import oracle.nosql.driver.ops.TableRequest;
+import oracle.nosql.driver.ops.TableResult;
+
+
+public class AlterTable{
+   /* Name of your table */
+   final static String tableName = "stream_acct";
+   static NoSQLHandle handle;
+   public static void main(String[] args) throws Exception {
+      //UNCOMMENT the lines of code below if you are using Oracle NoSQL Database Cloud service. Leave the lines commented if you are using onPremise database
```
Comment (mikebrey) [inline examples-nosql-java-sdk/sqlexamples/AlterTable.java:21]: same

File: examples-nosql-java-sdk/sqlexamples/AlterTable.java [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+      SignatureProvider ap = new SignatureProvider();
+      /* Create a NoSQL handle to access the cloud service */
+      NoSQLHandleConfig config = new NoSQLHandleConfig(region, ap);
+      // set your mycompartment
+      config.setDefaultCompartment(compId);
+      NoSQLHandle handle = NoSQLHandleFactory.createNoSQLHandle(config);
+      return handle;
+   }
+   /**
```
Comment (mikebrey) [inline examples-nosql-java-sdk/sqlexamples/AlterTable.java:62]: same

**Review by mikebrey — COMMENTED** [review]
please fix issues - since the same set of issues were throughout the code changes, I stopped reviewing.  However, address the issues throughout entire change.  Once done, pass back over for another review.

File: examples-nosql-node-sdk/sqlexamples/AlterTable.js [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+}
+/* Create and return an instance of a NoSQLCLient object for onPremises*/
+function getConnection_onPrem() {
+   /*replace the placeholder with your hostname*/
+   const kvstore_endpoint = `http://<hostname>:8080`;
+   return new NoSQLClient({
+      serviceType: "KVSTORE",
+      endpoint: kvstore_endpoint
+   });
+   /* if it is a secure store pass the credentials, username and password
```
Comment (mikebrey) [inline examples-nosql-node-sdk/sqlexamples/AlterTable.js:47]: This should probably use the UNCOMMENT type of comment, similar to what was in the java files.

**Review by mikebrey — COMMENTED** [review]
left a few more comments .  Please address globally.

**Comment by Vandana-Rajamani** [comment]
Addressed the comments for java and js files and changed all comments

File: examples-nosql-node-sdk/sqlexamples/AlterTable.js [hunk 8]
```diff
@@ -0,0 +1,88 @@
+/*Copyright (c) 2020, 2023 Oracle and/or its affiliates.
+ * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+ */
+'use strict';
+const NoSQLClient = require('oracle-nosqldb').NoSQLClient;
+const TABLE_NAME = 'stream_acct';
+
+/**
```
Comment (mikebrey) [inline examples-nosql-node-sdk/sqlexamples/AlterTable.js:8]: inconsistent comment formatting, please correct

File: examples-nosql-node-sdk/sqlexamples/CreateTable.js [hunk 9]
```diff
@@ -0,0 +1,72 @@
+/*Copyright (c) 2020, 2023 Oracle and/or its affiliates.
+ * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+ */
+'use strict';
+const NoSQLClient = require('oracle-nosqldb').NoSQLClient;
+const TABLE_NAME = 'stream_acct';
+/**
```
Comment (mikebrey) [inline examples-nosql-node-sdk/sqlexamples/CreateTable.js:7]: same

File: examples-nosql-python-sdk/sqlexamples/CreateTable.py [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+   #replace the placeholder with the ocid of your compartment
+   config.set_default_compartment("<ocid_of_your_compartment>")
+   return(NoSQLHandle(config))
+
+# Given a endpoint, instantiate a connection to the onPremise Oracle NoSQL database
+def get_connection_onprem():
+   #replace the placeholder with the name of your local host
+   kvstore_endpoint ='http://<hostname>:8080'
+   provider = StoreAccessTokenProvider()
+   #If using a secure store pass the username, password of the store to StoreAccessTokenProvider
```
Comment (mikebrey) [inline examples-nosql-python-sdk/sqlexamples/CreateTable.py:27]: inconsistent

**Review by mikebrey — COMMENTED** [review]
still seeing inconsistent comment formatting.   For a given language, format in those files should be consistent.

**Review by mikebrey — APPROVED** [review]

**Review by mikebrey — APPROVED** [review]

**Pull request merged**